util/assign.js

```diff
@@ -1,10 +1,4 @@
 // This exists so I'm only saving it once.
 "use strict"
 
-var hasOwn = require("./hasOwn")
-
-module.exports = Object.assign || function(target, source) {
-	for (var key in source) {
-		if (hasOwn.call(source, key)) target[key] = source[key]
-	}
-}
+module.exports = Object.assign
```

Comment:
Could you just drop this module entirely and use `Object.assign` directly in the places that use it?

Shouldn't impact performance, but will provide a small bundle size win *and* it's one less source file to worry about.

Reply:
OK, I removed the module.